### ISSUE TITLE
Why is there no test_loss

### ISSUE THREAD

**Issue by Zzz-L** [jonnykira/Tensorflow_mLSTM#7]
Hi @jonnykira, thanks very much for sharing this!
why is no test_loss in train_mLSTM, only find the train_loss

**Comment by jonny-d** [comment]
The code only trains a language model on the training set, if you would like to split your data into training/test set and measure loss on the test set, it should be fairly straightforward to modify the code.

**Comment by Zzz-L** [comment]
I mean，Don't you need to go through the test loss evaluation language model，can you just use the train loss to measure the model ?  beacuse from the  paper  "Generating Reviews and Discovering Sentiment." , there need split data into training/test set.
look forward to your reply , thanks
